cmd/rookd/operator.go

```diff
@@ -35,19 +35,12 @@ https://github.com/rook/rook`,
 }
 
 func init() {
-	operatorCmd.Flags().StringVar(&cfg.containerVersion, "container-version", "latest", "version of the rook container to launch")
-	operatorCmd.Flags().StringVar(&cfg.namespace, "namespace", "", "the namespace in which the operator is running")
```

Comment:
is the rook-operator.yaml, they would need to specify the `namespace` attribute in the metadata section.